static/app/components/activity/item/index.tsx

```diff
@@ -26,43 +23,38 @@ interface ActivityItemProps {
     type: ActivityAuthorType;
     user?: AvatarUser;
   };
-  // Size of the avatar.
   avatarSize?: number;
   bubbleProps?: ActivityBubbleProps;
-
-  children?: React.ReactChild | ChildFunction;
```

Comment:
We were only doing this to allow rendering without padding, added `noPadding` for that